lib/data_layer/models/nip_05_model.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:ndk/domain_layer/entities/nip_05.dart';
+
+/// model for converting json <==> dart object
+class Nip05Model extends Nip05 {
```

Comment:
Why is this necessary?

Reply:
we can remove this for now. Its could be used to convert from and to json. But we are not using it right now. 
Maybe this canges if we save in db so I would like to leave it for now

Reply:
Can't we have `.fromJson` and `.toJson` in the `Nip05` class?
Analogously to for example `Metadata.fromJson` ?

Reply:
yes its just cleaner, no dependencies in the entity